src/main/java/io/kestra/plugin/youtube/Triggers/CommentTrigger.java

```diff
@@ -0,0 +1,248 @@
+package io.kestra.plugin.youtube.Triggers;
+
+import com.google.api.client.auth.oauth2.BearerToken;
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.api.services.youtube.YouTube;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.conditions.ConditionContext;
+import io.kestra.core.models.executions.Execution;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.models.triggers.*;
+import io.kestra.core.runners.RunContext;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Trigger on new youtube comments",
+    description = "Monitors youtube videos for new comments and triggers executions when found."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Monitor video for new comments",
+            full = true,
+            code = """
+                id: youtube_comment_monitor
+                namespace: company.team
+
+                tasks:
+                  - id: notify_slack
+                    type: io.kestra.plugin.notifications.slack.SlackIncomingWebhook
+                    url: "{{ secret('SLACK_WEBHOOK_URL') }}"
+                    payload: |
+                      {
+                        "text": "New comment from {{ trigger.authorDisplayName }}: {{ trigger.textDisplay }}"
+                      }
+
+                triggers:
+                  - id: new_comment_trigger
+                    type: io.kestra.plugin.youtube.triggers.CommentTrigger
```

Comment:
Put triggers in the same package than `VideoStats` and update to the correct package in all examples 🙏